models/forgefed/branch.go

```diff
@@ -0,0 +1,55 @@
+// Copyright 2022 The Gitea Authors. All rights reserved.
```

Comment:
I think you could move the files to `modules/forgefed` if they are not related with database operations.